03 BusBoard/BusBoard/BusBoard/Models/Arrival.cs

```diff
@@ -0,0 +1,55 @@
+using System.Text.Json.Serialization;
+
+namespace BusBoard.Models;
+
+public class Arrival
+{
+    [JsonPropertyName("lineName")]
+    public required string LineName {
+        get;
+        set; 
+    }

```

Comment:
minor nit but you could refactor this to be a bit more consise
```suggestion
    public required string LineName { get; set; }
```